site/overview/faqs.md

```diff
@@ -0,0 +1,154 @@
+
+We recommend adding error handling when sending a `UserOperation` to handle potential gas and fee changes due to market movement. Learn more about [frequent errors](#common-errors).
+:::
+
+## Gas Manager
+
+### What tiers support gas sponsorship?
+
+::: details Answer
+Gas sponsorship is available on testnet for all tiers. For support on mainnet, you must be on a paying tier (i.e. Growth tier and above). Learn more about our different pricing tiers [here](https://docs.alchemy.com/reference/gas-manager-coverage-api-pricing#fee-logic).
+:::
+
+### How is gas sponsored? Do I need to fund the gas manager?
+
+::: details Answer
+We front the gas for your application and put the USD equivalent on your bill at the end of the month. No need to worry about pre-funding the gas manager or conversions, we’ve got you covered!
+:::
+
+### What are my gas sponsorship limits?
+
+::: details Answer
+You can find details of gas manager limits depending on your tier [here](https://docs.alchemy.com/reference/gas-manager-coverage-api-pricing#fee-logic).
+:::
+
+### Do you support ERC20 or stablecoin paymasters?
+
+::: details Answer
+Currently, we don’t support this, but we are actively exploring. Please [reach out](/overview/contact-us) if you are interested as we would love your input in our spec.
+:::
+
+### How is the gas manager protected from DDOS attacks?
+
+::: details Answer
+In your gas manager policy, you can configure spending rules per address, per app, and/or policy wide limits. See how to set up these policies [here](/tutorials/sponsoring-gas/sponsoring-gas.html#_2-create-a-gas-manager-policy).
+:::
+
+## Common Errors {#common-errors}
+
+### Invalid policy ID: `{ code: -32602, message: 'Invalid Policy ID' }`
+
+::: details Answer
+Gas manager policies can only be tied to one app. Make sure you are using the app key id that is associated with the app the Gas Manager policy is configured for or create a new policy for the app you are using.
+:::
+
+### Precheck failed: `{ code: -3200, message: 'precheck failed: ...' }`
+
+::: details Answer
+Precheck failed errors are often related to gas and/or fees. Our bundler follows standard [ERC 4337](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4337.md#client-behavior-upon-receiving-a-useroperation) implementation for gas and fee checks in order to 1) ensure your user operations land on chain and to 2) protect the bundler from potential attacks in order to support scalability.
+
+These errors are often related to market movement between the time gas and fees are estimated and the time when user operations are submitted to the bundler. This fluctuation in the market is especially variant on testnet. We currently reject upon sending if the user operation is underpriced compared to the network rate to ensure your user operation is included in a block.
```

Comment:
since UO spelled out above
```suggestion
These errors are often related to market movement between the time gas and fees are estimated and the time when UOs are submitted to the bundler. This fluctuation in the market is especially variant on testnet. We currently reject upon sending if the user operation is underpriced compared to the network rate to ensure your UO is included in a block.
```